scripts/build.js

```diff
@@ -69,11 +74,19 @@ if (!fs.existsSync(treeSitterCli)) {
     let command
     if (process.env.CI) {
       console.log(`Compiling ${module}`)
-      command = `node_modules/.bin/tree-sitter build-wasm ${module}`
+      if (module.endsWith('tree-sitter-php')) {
+        command = `node_modules/.bin/tree-sitter build --wasm ${module}/php`
```

Comment:
Why does php get an exception here?

Reply:
The new version of php is multi grammar it seems. It's got option of php (php + html) and php_only (just php).